selfdrive/car/chrysler/carcontroller.py

```diff
@@ -49,12 +49,17 @@ def update(self, CC, CS, now_nanos):
 
       # TODO: can we make this more sane? why is it different for all the cars?
       lkas_control_bit = self.lkas_control_bit_prev
-      if CS.out.vEgo > self.CP.minSteerSpeed:
+      if self.CP.carFingerprint in RAM_DT:
+        if CS.out.vEgo >= self.CP.minEnableSpeed and CS.out.vEgo <= self.CP.minEnableSpeed + 0.5:
+          lkas_control_bit = True
+        if (self.CP.minEnableSpeed >= 14.5)  and (CS.out.gearShifter != car.CarState.GearShifter.drive) :
```

Comment:
```suggestion
        if (self.CP.minEnableSpeed >= 14.5) and (CS.out.gearShifter != 2):
```
Let's use the cereal ordinal here so one less import.